Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -75,8 +75,42 @@ Platform Platform
 
 		FragmentManager FragmentManager => _fragmentManager ?? (_fragmentManager = ((FormsAppCompatActivity)Context).SupportFragmentManager);
 		bool IsBottomTabPlacement => (Element != null) ? Element.OnThisPlatform().GetToolbarPlacement() == ToolbarPlacement.Bottom : false;
-		public Color BarItemColor => (Element != null) ? Element.OnThisPlatform().GetBarItemColor() : Color.Default;
-		public Color BarSelectedItemColor => (Element != null) ? Element.OnThisPlatform().GetBarSelectedItemColor() : Color.Default;
+
+		public Color BarItemColor
+		{
+			get
+			{
+				if (Element != null)
+				{
+					if (Element.IsSet(TabbedPage.UnselectedTabColorProperty))
+						return Element.UnselectedTabColor;
+
+#pragma warning disable CS0618 // Type or member is obsolete
+					return Element.OnThisPlatform().GetBarItemColor();
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
Actually, I don't think we need to do this. Because we are setting the new property via the old one, this check becomes redundant.